modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2175,14 +2181,15 @@
 		end
 	end
 
-
 	function m.programDatabaseFile(cfg)
 		if cfg.symbolspath and cfg.symbols ~= p.OFF and cfg.debugformat ~= "c7" then
-			m.element("ProgramDatabaseFile", nil, p.project.getrelative(cfg.project, cfg.symbolspath))
+
+			local key = iif(cfg.kind == p.STATICLIB, "ProgramDataBaseFileName", "ProgramDatabaseFile")
```

Comment:
Personally, I would prefer this to be split into two methods, one for "ProgrameDatabaseFileName", and one for "ProgramDatabaseFile"..

While I get that they will be almost identical except for their name, it allows for easier override functionality...

Reply:
That was my first implementation but that would duplicate the condition and would be a pain to maintain IMO.
But I get your point about override, if you feel that's the way to go, I have no objection.
I will make the changes later today.